README.md

```diff
@@ -132,6 +132,7 @@ Focus is on the specific task.
 ## Security
 *Plugins and information around security, preventing vulnerabilities and protection against XSS and alike.*
 
+- [Xcrypto](https://github.com/Simbacode/xcrypto) -  Allows excyption of request to and from cakephp server using DES,AES,PHP,HASH,RC4,RSA,Rijndael,TripleDES
```

Comment:
Hm, looking at it, isn't that just a normal PHP lib? What makes this a CakePHP specific tool?